third_party/paddle/BUILD

```diff
@@ -0,0 +1,832 @@
+load("@rules_cc//cc:defs.bzl", "cc_import", "cc_library")
+
+package(
+    default_visibility = ["//visibility:public"],
+)
+
+output_files = [
+    "paddle/include/paddle_inference_api.h",
```

Comment:
Can these files lists be replaced by fileglob???

Reply:
Glob only searches files in its own package, and looks only for source files (**not generated files** nor other targets).